spec/menu_spec.rb

```diff
@@ -2,12 +2,16 @@
 
 RSpec.describe Menu do
 
-  let(:mains) { Menu::MAINS }
-  let(:list) { subject.send(:sides_list) }
-
-  describe '#list' do
+  describe '#menu' do
     it 'prints a full list of available dishes' do
-      expect(subject.show).to include list
+      expect{subject.show}.to output.to_stdout
```

Comment:
Maybe stub the menu and expect to output the contents of the stub? output.to_stdout feels like it would always pass.